assets/app/scripts/directives/logViewer.js

```diff
@@ -268,8 +286,45 @@ angular.module('openshiftConsole')
                   });
                 });
               });
+
+
+
+              // scrollable node is window if mobile, else a particular DOM node.
+              var setScrollable = function() {
```

Comment:
Yeah, i guess the link fn is doing the finding with the `document.getElementById` & `cacheLogNode `.  So `set`, `toggle`, `find`... not sure what the best word here is?